wordpress-mobile/WordPress-Android
# Fix v13.0 editors opt-in migration strategy problems

This PR improves and fixes the editors opt-in migration strategy by changing a couple of important things:

- There is now ONE rest call only, that sets the editor preference for all sites
- There is no need to synch editor settings if the local app-wide flag is still set in the app and the migration still need to happens

To test:
- Install 12.9, and turn block editor to ON
- Install 13.0 and check that everything is fine since the first launch

This still references a FluxC hash, we need to switch to a tag when the FluxC companion PR is merged and properly tagged.

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
